subsys/bluetooth/mesh/gen_plvl_srv.c

```diff
@@ -447,16 +447,21 @@ static void lvl_delta_set(struct bt_mesh_lvl_srv *lvl_srv,
 	struct bt_mesh_plvl_srv *srv =
 		CONTAINER_OF(lvl_srv, struct bt_mesh_plvl_srv, lvl);
 	struct bt_mesh_plvl_status status = { 0 };
-
-	uint16_t start_value = srv->last;
+	uint16_t start_value;
```

Comment:
One thing that confuses me is the use of start and start_value. Is there any difference between them. Also I think "value" should be avoided if it's not generic, if there's no reason I'm unaware of. Maybe change to start_lvl to differentiate from temperature etc.

Reply:
There's no difference, so it's an unnecessary inconsistency. I'll change it.